plugins/docker/docker.plugin.zsh

```diff
@@ -0,0 +1,42 @@
+alias di='docker info'
+alias dlg='docker container logs'
+alias dls='docker container ls'
+alias dlsa='docker container ls -a'
+alias dr='docker container run'
+alias drit='docker container run -it'
+alias drm='docker container rm'
+alias drmf='docker container rm -f'
```

Comment:
I like the `!` suffix notation for force-commands. I worry a typo can delete the wrong container by accident:
```suggestion
alias 'drm!'='docker container rm -f'
```

Reply:
Done!